apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -1,19 +1,28 @@
 import {
 	Loader,
 	Notification,
+	QuizQuestion,
 	RouterOutlet,
 } from "~/libs/components/components.js";
 import { AppRoute, DataStatus } from "~/libs/enums/enums.js";
 import {
 	useAppDispatch,
+	useAppForm,
 	useAppSelector,
 	useEffect,
 	useLocation,
 } from "~/libs/hooks/hooks.js";
 import { actions as userActions } from "~/modules/users/users.js";
 
+type FormValues = {
+	value: string;
+};
+
 const App: React.FC = () => {
 	const { pathname } = useLocation();
+	const { control } = useAppForm<FormValues>({
```

Comment:
Create a separate quiz component and handle all logic there

Reply:
Should quiz question component be in folder with new quiz component(quiz component -> components -> quiz question component) or in separate folders?

Reply:
Should be in ~/pages/quiz/libs/components

Reply:
Should I add mock data?

Reply:
We already have PR for adding questions, consult with that developer to get a real type. You can mock data for now, but use a real type